content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+Documentation
+#############
+
+:menu_order: 007
+
+.. contents::
+
+
+General
+=======
+
+**What is documented?**
+
+**For the Chevah project members:**
+
+* Packages, modules, classes and functions using docstrings. 
```

Comment:
Maybe 

Use docstrings for all the code (python/shell/c..etc)

but I don't understand this section...

it starts with "What is documented" and then it talks about linking certain version of STPPlus... this does not look like a "what" but "how"